drltrace/drltrace.cpp

```diff
@@ -98,6 +369,15 @@ print_simple_value(drsys_arg_t *arg, bool leading_zeroes)
 }
 
 static void
+print_string(void *pointer_str, bool is_wide)
+{
+    if (pointer_str == NULL)
+        dr_fprintf(outf, "<null>");
+    else
+        dr_fprintf(outf, is_wide ? "%S" : "%s", pointer_str);
```

Comment:
There's no existing convenience routine: you'd have to use a DR try-except or write a loop that uses a safe read.